### PR TITLE
Mention the checkup period of two factor auth

### PR DESCRIPTION
**Pull request by Kakulukian**
I think it's clear?

### CODE REVIEW

File: docs/hub/security-two-fa.md [hunk 1]
```diff
@@ -35,6 +35,10 @@ In Hugging Face Hub:
 
 If you entered the correct pin, the Hub displays a list of recovery codes. Download them and keep them in a safe place.
 
+<Tip>
+After configuring 2FA, your account enters a 30-day check-up period. To exit this period, successfully complete 2FA within those 30 days. If you do not complete 2FA during this time, you will be prompted to perform 2FA in an existing session on the 30th day.
```
Comment (SBrandeis) [inline docs/hub/security-two-fa.md:39]: A bit convoluted IMO, maybe we can do something simpler

Would this be a correct assertion?
```suggestion
You will be prompted for 2FA on every new session, and every 30 days for active sessions.
```

Comment (Kakulukian) [reply docs/hub/security-two-fa.md:39]: Yes it is

Comment (SBrandeis) [reply docs/hub/security-two-fa.md:39]: "session" might not be the clearest wording here

Comment (coyotte508) [reply docs/hub/security-two-fa.md:39]:
```suggestion
You will be prompted for 2FA every time you log in, and after that every 30 days.
```

Comment (coyotte508) [reply docs/hub/security-two-fa.md:39]: or cc @Michellehbn x)

Comment (julien-c) [inline docs/hub/security-two-fa.md:39]:
```suggestion
You will be prompted for 2FA every time you log in, and every 30 days.
```

**Review by julien-c — APPROVED** [review]
did a mix of the two formulations, LGTM like this

**Comment by julien-c** [comment]
also please consider merging https://github.com/huggingface/hub-docs/pull/1217 too

**Pull request merged**